plutus-use-cases/src/Language/Plutus/Coordination/Contracts/Swap.hs

```diff
@@ -0,0 +1,142 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE RecordWildCards   #-}
+-- Disabled until we can use the Num.(*) for `Ratio Int`
```

Comment:
Hm, is there a bug for this?

Reply:
Wasn't sure whether this was expected behaviour or not. But here is a ticket anyway: https://iohk.myjetbrains.com/youtrack/issue/CGP-300